packages/orderbook/test/js_api/addOrder.test.ts

```
@@ -368,14 +368,8 @@ describe('Rain Orderbook JS API Package Bindgen Tests - Add Order', async functi
 			.thenReply(200, JSON.stringify({ data: { addOrders: addOrders } }));
 
-		try {
-			const result: SgAddOrderWithOrder[] = await getTransactionAddOrders(
-				mockServer.url + '/sg1',
-				transaction1.id
-			);
-			assert.equal(result[0].order.id, mockAddOrder.order.id);
-		} catch (e) {
-			assert.fail('expected to resolve, but failed' + (e instanceof Error ? e.message : String(e)));
-		}
+		const result = await getTransactionAddOrders(mockServer.url + '/sg1', transaction1.id);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)